array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-unused-vars */
+function compact(array){
+    let newArr = [];
+    for(let i = 0;i<array.length;i++){
+        if(!array[i]){}
+        else{
+            newArr.push(array[i]);
+        }

```

Comment:
You can simplify this by just testing if `array[i]` is truthy. No `else` needed.

```js
if (array[i]) {
  // push it
}
```